### DISCUSSION TITLE
请问筹款时交易发送只有transact to FundMe.fund pending ... 的原因

### DISCUSSION THREAD

**Discussion by DonaldAI** [smartcontractkit/Web3_tutorial_Chinese#7] [General]
按照相同的代码和操作进行，小于Min值时没有提示gas估计，然后使用40finney时，只提示transact to FundMe.fund pending ... ，metamask没有响应，请问老师这可能是什么原因？应该如何调整呢？谢谢
![微信截图_20241026170942](https://github.com/user-attachments/assets/7f2b8bea-0a12-4d1e-a1da-d622a40c691d)

**Comment by QingyangKong** [comment]
这个是因为在 remix 中没有成功唤起 metamask 插件，一般刷新一下页面就可以解决。

**Comment by DonaldAI** [comment]
您的邮件已收到！
